src/components/elements/Inputs/FileInput/FileInput.tsx

```diff
@@ -117,30 +125,54 @@ const FileInput = (props: FileInputProps) => {
             multiple={props.allowMultiple}
             accept={accept}
           />
-          <div className="m-auto flex w-fit items-center justify-center gap-3">
-            <FileCardContent
-              title={labelText}
-              subtitle={`${t("Drag and drop or")} <span className="text-primary underline">${t(
-                "browse your device"
-              )}</span>`}
-              thumbnailClassName="fill-primary"
-              thumbnailContainerClassName="bg-primary-100"
-            />
-          </div>
+          <When condition={!variant.snapshotPanel}>
+            <div className="m-auto flex w-fit items-center justify-center gap-3">
+              <FileCardContent
+                title={labelText}
+                subtitle={`${t("Drag and drop or")} <span className="text-primary underline">${t(
+                  "browse your device"
+                )}</span>`}
+                thumbnailClassName="fill-primary"
+                thumbnailContainerClassName="bg-primary-100"
+              />
+            </div>
+          </When>
+          <When condition={variant.snapshotPanel}>
+            <Icon name={IconNames.UPLOAD_CLOUD} className="mb-4 h-5 w-5" />
+            <div className="flex flex-col">
+              <Text variant="text-12-bold" className="text-center text-primary">
+                Click to upload
+              </Text>
+              <Text variant="text-12-light" className="text-center">
+                or
+              </Text>
+              <Text variant="text-12-light" className="max-w-[210px] text-center">
+                {props.descriptionInput}
+              </Text>
+            </div>
+          </When>
         </div>
       </InputWrapper>
+      <When condition={variant.listPreviewDescription}>
+        <div className={variant.listPreviewDescription}>
+          {props.descriptionList}
+          <Text variant="text-12-bold" className="mt-9 pr-5 text-primary">
+            {props.descriptionListStatus}
```

Comment:
I'm not sure about translation for these two. I think the List doesn't need it because it's a react node, but the status does.

Reply:
done add t for descriptionListStatus